doc/source/releases/17.01.rst

```diff
@@ -9,163 +9,124 @@
-  (thanks to `@mvdbeek <https://github.com/mvdbeek>`__).
-  `Pull Request 3106`_
-* update rename dataset action to accept . in repeat input parameters
+* Add ESTScan scores matrices (smat) datatype
+  (thanks to `@gregvonkuster <https://github.com/gregvonkuster>`__).
+  `Pull Request 3258`_
+* Improve GG datatype sniffer to sniff fewer texts files as GG
+  `Pull Request 3233`_
+* Allow rename dataset post job actions to work on input parameters inside of ``repeat`` blocks
   (thanks to `@scholtalbers <https://github.com/scholtalbers>`__).
   `Pull Request 3197`_
-* Tour routing tweaks, allows us to drop the model out of menu
+* Improve Tour URL routing in the client.
   `Pull Request 3214`_
-* Replace CasperJS browser tests with Selenium tests.
-  `Pull Request 3239`_
-* 3 enhancements for the SLURM job runner
+* Assorted enhancements for the SLURM job runner
   (thanks to `@nsoranzo <https://github.com/nsoranzo>`__).
   `Pull Request 3218`_
-* Add EDAM topic(s) and operation(s) elements to XSD file.
-  (thanks to `@hmenager <https://github.com/hmenager>`__).
-  `Pull Request 3221`_
-* Further tighten up GG format.
-  `Pull Request 3233`_
-* change conda channel order
-  (thanks to `@bgruening <https://github.com/bgruening>`__).
-  `Pull Request 3237`_
-* Sort environment variables in job info page
-  (thanks to `@erasche <https://github.com/erasche>`__).
-  `Pull Request 3256`_
-* Add support for ESTScan scores matrices (smat) data type
-  (thanks to `@gregvonkuster <https://github.com/gregvonkuster>`__).
-  `Pull Request 3258`_
-* Increase release delta to 4 months ahead of 17.05.
+* Various Galaxy tool schema definition fixes (including adding
+  EDAM topic(s) and operation(s) thanks to `@hmenager <https://github.com/hmenager>`__).
+  `Pull Request 3221`_, `Pull Request 3216`_
+* Formalize decision to increase time between releases to four months. The next release
+  will be 17.05.
   `Pull Request 3271`_
-* Small touchups to run_tests.sh.
-  `Pull Request 3285`_
-* Slim down testing ``Dockerfile``.
-  `Pull Request 3286`_
-* Ready Integration Testing for Prime Time
-  `Pull Request 3291`_
-* lexists vs. exists
+* Numerous improvements to the test framework including replacing broken CasperJS UI tests
+  with newer Selenium based ones and expanded test coverage with a particular
+  focus on web and integration testing.
+  `Pull Request 3285`_, `Pull Request 3286`_, `Pull Request 3291`_,
+  `Pull Request 3342`_, `Pull Request 3367`_, `Pull Request 3381`_,
+  `Pull Request 3239`_, `Pull Request 3371`_, `Pull Request 3190`_,
+  `Pull Request 3264`_, `Pull Request 3288`_, `Pull Request 3387`_,
+  `Pull Request 3400`_, `Pull Request 3401`_, `Pull Request 3228`_,
+  `Pull Request 3365`_, `Pull Request 3368`_, `Pull Request 3370`_,
+  `Pull Request 3482`_, `Pull Request 3185`_
+* Fix to use lexists instead exists in dependency resolver code
   (thanks to `@bgruening <https://github.com/bgruening>`__).
   `Pull Request 3301`_
```

Comment:
I actually moved this into the bug fixes section. I think that it is fine down there. We could list it under "misc" fixes for dependency resolvers also - do you feel strongly about this?